compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/checkers/declaration/FirFunctionParameterChecker.kt

```diff
@@ -70,4 +78,46 @@ object FirFunctionParameterChecker : FirFunctionChecker() {
             }
         }
     }
+
+    private fun checkUninitializedParameter(function: FirFunction<*>, context: CheckerContext, reporter: DiagnosticReporter) {
```

Comment:
nit: can we rename this function to `checkUninitializedParameterWithEscapedUnderscoreAsName`?

Reply:
More specific, I like it, thanks!